samples/python/qrcode.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import print_function
+import cv2 as cv
+import numpy as np
+import sys
+from glob import glob
+PY3 = sys.version_info[0] == 3
+import argparse
+
+if PY3:
+    xrange = range
+
+def str2bool(v):
```

Comment:
Use `action='store_true'` in `.add_argument()` instead. Refer to other code in OpenCV repository.